README.md

```diff
@@ -16,6 +21,7 @@
 
 3. Install project dependencies:
 
+  ```  yarn install
```

Comment:
hey @carloslancha, This is breaking the code blocks below.

Reply:
Don't know what's that doing there, thx!